src/backend/providers/playnite/PlayniteProvider.cpp

```diff
@@ -114,6 +114,17 @@ void apply_game_fields(
             QStringLiteral("%1/library/files/%2").arg(playnite_dir_path, game_info.background_image);
         game.assetsMut().add_file(AssetType::BACKGROUND, bg_path);
     }
+
+    const QString extra_metadata_path = QStringLiteral("%1/ExtraMetadata/games/%2").arg(playnite_dir_path, game_info.id);
```

Comment:
`game_info.id` could be empty here.

Reply:
Good catch, added validation for game ID